source/help/getting-started/organizing-conversations.rst

```diff
@@ -154,6 +154,8 @@ When configured by the System Admin, users can enable the features in **Account
 
 **Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.
 
+**Move channels and categories**: Prioritize important conversations by dragging and dropping channels between or within categories, or drag channels to reorder entire categories. You can also select the **More options** icon, then select **Move to**, followed by a category destination.
```

Comment:
There should be a mention that this is only available in Cloud and will be available for on-prem later in v5.32.

Reply:
Agreed! Thanks! Worked with the PMs to define a process for such cases. The documentation now clarifies when the feature will be available for self-hosted deployments.